packages/shared/src/components/sidebar/SidebarUserButton.tsx

```diff
@@ -28,14 +29,16 @@ export function SidebarUserButton({
         <li className="flex flex-col p-6 pt-2">
           {user && user?.infoConfirmed ? (
             <>
-              <div className="flex justify-between items-center mb-4">
+              <div className="flex items-center mb-4">
                 <ProfileLink
                   href={user.permalink}
                   className="flex items-center p-0 ml-0.5 font-bold no-underline rounded-lg border-none cursor-pointer text-theme-label-primary bg-theme-bg-secondary typo-callout focus-outline"
                 >
                   <ProfilePicture user={user} size="medium" />
                   <span className="mr-3 ml-2">{user.reputation ?? 0}</span>
                 </ProfileLink>
+                <SearchReferralButton className="ml-3" />
```

Comment:
This should be conditional, right? Only when the user has the placement for the header since they can still get the option in the profile menu.

Reply:
I moved that condition inside the component as its now re-used 👍

Reply:
Oh, right. In that case, we should be good here :shipit: